### ISSUE TITLE
Remote debugger networking interface

### ISSUE THREAD

**Issue by FredrikNoren** [AmbientRun/Ambient#463]
We should start moving towards a remote debugger networking interface. I'm thinking we set up a separate port, which a debugger UI can then connect to. Some of the things we'll want to start adding/migrating to this interface:

- [ECS debugger](https://github.com/AmbientRun/Ambient/tree/main/crates/ecs_editor)
- Asset cache timeline ([asset timeline](https://github.com/AmbientRun/Ambient/tree/main/crates/asset_timeline) already exists)

**Comment by pierd** [comment]
Are there any useful tools based on `rcon` or other protocols? It might be handy to support some industry standard to leverage what's already out there.

**Comment by philpax** [comment]
Nah, it's all pretty bespoke I think - each game/engine has different requirements
